scripts/delete-inactive-helm-releases.sh

```diff
@@ -45,12 +52,23 @@ for ns in $(echo ${!namespaceMapping[*]}); do
       fi
       cutoff=$((cutoffDays*24*3600))
       if [[ $((currenttime-lastUpdated)) -gt "$cutoff" && ${releaseName} = "*-pr-*" ]]
-       then
-         echo "Deleting helm release ${releaseName} as it is inactive for more than ${cutoffDays} days. Last updated : ${date} "
-         helm delete --namespace "${ns}" "${releaseName}"
-#         Enable for debug if needed
-#        else
-#          echo "Skipping ${releaseName} as it is not inactive for ${cutoffDays}, Last updated: ${date}, cutoff=${cutoff}, result=$((currenttime-lastUpdated))"
+      then
+          pr=$(echo "${releaseName##*-pr-}")
+          repo=${repos[$(echo "${releaseName%-pr-*}")]}
+
+          labels=$(curl -s https://api.github.com/repos/hmcts/${repo}/issues/${pr} | jq -r '.labels[]?.name')
```

Comment:
could we use the CLI?
https://cli.github.com/manual/gh_pr_view

Reply:
Possibly, but I'd rather not add dependencies to other libraries if at all possible.

Reply:
After further feedback, there's one additional point here: the `curl` call will execute regardless also for repositories that don't exist. This will cause us to meet a throttling limitation very quickly. I'll be working to apply the `curl` call only to the repositories that are explicitly mentioned.